python/taichi/README.md

```diff
@@ -267,7 +267,7 @@ Embedding the language in `python` has the following advantages:
 - Easy to learn. Python itself is very easy to learn, so is PyTaichiLang.
 - Easy to run. No ahead-of-time compilation is needed.
 - It allows people to reuse existing python infrastructure:
-   - IDEs. A python IDE simply works for TaichiLang, with syntax highlighting, checking, and autocomplete.
+   - IDEs. A python IDE simply works for TaichiLang, with syntax highlighting, checking, and autocomplete. For taichi developers, [ffi-navigator](https://github.com/tqchen/ffi-navigator) allows you to jump from C++ extension symbols exported via taichi_core in Python code to their defintion in C++.
```

Comment:
The [Contributor Guide](https://taichi.readthedocs.io/en/latest/contributor_guide.html#tips-on-taichi-compiler-development) with source at
https://github.com/taichi-dev/taichi/blob/master/docs/contributor_guide.rst#tips-on-taichi-compiler-development seems to be a better place for this tip :-)

Reply:
`python/taichi/README.md` is a very old introduction to Taichi. I should have removed this file. Sorry about the confusion!

Reply:
Updated.